system/jlib/jstats.cpp

```diff
@@ -122,16 +122,16 @@ MODULE_INIT(INIT_PRIORITY_STANDARD)
 
 extern jlib_decl int compareScopeName(const char * left, const char * right)
 {
-    if (!*left)
+    if (!left || !*left)
     {
-        if (!*right)
+        if (!right || !*right)
```

Comment:
use isEmptyString() ?

Reply:
I could do.  In this case I think I prefer explicitly checking for null.